tests/compat.sh

```diff
@@ -13,6 +13,10 @@
 # Check each common ciphersuite, with each version, both ways (client/server),
 # with and without client authentication.
 
+# Limit log output to 10 GB, in case of failures with this script where it may
```

Comment:
`ulimit -f` limits the size of each file, but this sentence reads to me like it's limiting the total size of logs. Please change to e.g. “Limit the size of each log to 10 GB”.